python/triton_kernels/tests/test_reduce.py

```diff
@@ -59,10 +60,15 @@ def plus_a_reduce(x, a):
 ])
 @pytest.mark.parametrize("dim", [0, 1, 2])
 def test_op(B, M, N, dtype_str, dim, mask_mode, postprocess_fn):
-    is_hip = triton.runtime.driver.active.get_current_target().backend == "hip"
-    is_pre_h100 = torch.cuda.get_device_capability() < (9, 0)
-    if (is_hip or is_pre_h100) and "float8" in dtype_str:
-        pytest.skip("float8 not supported on CUDA < 9.0")
+    # Check float8 hardware support
+    is_pre_h100 = torch.cuda.get_device_capability() < (9, 0) if torch.cuda.is_available() else False
```

Comment:
`is_cuda() and torch.cuda.get_device_capability() < (9, 0)`. `torch.cuda.is_available()` returns true for AMD too. (And torch.cuda.get_device_capability() also returns numbers like `(9, 5)` for gfx950 IIRC.)